### DISCUSSION TITLE
weather/ cloud overlay? good for biker (bi/motor- cyclists)/ outdoor/ traveling

### DISCUSSION THREAD

**Discussion by flobee** [osmandapp/OsmAnd#14032] [Ideas]
Hi
i saw that garmin zumo xt already features cloud/ weather informations as "overlay".

German: 
```
"
**Apps**
...
Live Verkehrsdaten und sogar ein Regenradar sind an Bord und sehr praktisch auch für Biker....
https://www.motorradreisefuehrer.de/Garmin_Zumo_XT.html
"
```

Does a feature like this already exists?
 Well. For weather i use the specialised german version from DWD (Deutscher Wetter Dienst) which is the best (but only for germany)

btw: I still have a problem using osmand on mobil phones because of the display when the sun is shining. (some garmin navi products solve this very very good). 

Any hints what mobile device i could use?

 I still remember the very first mobil phones (not smart phones) with the LCD displays where the contrast gets better when the sun was shining (black/wihite and 4c displays. very old fashing but works perfect for navigation & outdoor).

Kind reards
Florian

**Comment by mariush444** [comment]
1. cloud/ weather informations as "overlay".
If you know WMS/WMST (x/y/z and not lat/lon) service for weather then it can be use as a overlay in osmand
I see that https://test.osmand.net use something like Ventusky so maybe it will be implemented in osmand app.
2. LCD is as good as you bought. AMOLED is probably better than IPS for your needs.
Some android UI has possibility to set/modify colors to be more contrast.

**Comment by flobee** [comment]
btw. [openweathermap](https://openweathermap.org/weathermap?basemap=map&cities=true&layer=clouds) does it since... ages i think.

**Comment by mariush444** [comment]
so...  it is not the best but what I can do.
this is .metainfo file for temperature

xxxx - you should get your own appid from openweather
then you can see color tiles for temp. (yellow, green, blue, and so on) but you have to know/remember which color means what temp in degrees.

[url_template]
http://a.sat.owm.io/vane/2.0/weather/TA2/{0}/{1}/{2}?appid=xxxxxx&fill_bound=true
[ext]
.jpg
[min_zoom]
1
[max_zoom]
18
[tile_size]
256
[img_density]
16
[avg_img_size]
20000

and the clouds

[url_template]
http://a.sat.owm.io/vane/2.0/weather/CL/{0}/{1}/{2}?appid=xxxx
[ext]
.jpg
[min_zoom]
1
[max_zoom]
18
[tile_size]
256
[img_density]
16
[avg_img_size]
20000

the precipitation is a problem because as a parameter the date and time is needed and it is not possible to do it via osmand UI.

**Comment by mariush444** [comment]
one correction because I noticed that some set of servers are used:
http://a.sat.owm.io , http://b.sat.owm.io , http://c.sat.owm.io ...

then url should be change to 
http://{rnd}.sat.owm.io

and the block should be added in .metainfo file
[randoms]
a,b,c,d,e,f,g,h

**Reply by flobee** [reply]
umpf. Nerdish by nature. :-) 

`{rnd}` what? `.metainfo` where?

sry. with osmand i'm on the user level, not implementation details.

**Comment by mariush444** [comment]
https://maptile.osmand.net/docs/technical/osmand-file-formats/osmand-metainfo/

it's not "black magic" in my opinion

**Reply by flobee** [reply]
ah ok!
